components/match2/commons/match_group_util.lua

```diff
@@ -648,6 +674,27 @@ function MatchGroupUtil.indexTableToRecord(coordinates)
 	end)
 end
 
+-- Convert 1-based indexes to 0-based
+function MatchGroupUtil.indexTableToRecord(coordinates)
```

Comment:
This function is already in the codebase?

Reply:
Yeah it was already added